### PR TITLE
모델 D (single frame double backbone) 에 focal loss 추가

### PR DESCRIPTION
**Pull request by keechang-choi**
focal loss 이동 및 추가

### CODE REVIEW

**Review by dozen12 — APPROVED** [review]
LGTM

File: src/models/cnn_2_5/version/B/model.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 from config import CFG
 import torchmetrics
 
-from utils.mcc import MCC_Loss
+from utils.loss import MCC_Loss
```
Comment (dozen12) [inline src/models/cnn_2_5/version/B/model.py:9]: @keechang-choi  여기 이상함

File: src/models/cnn_2_5/version/C/model.py [hunk 2]
```diff
@@ -5,10 +5,9 @@
 import timm
 from config import CFG
 import torchmetrics
-import torchvision
-
-from utils.mcc import MCC_Loss
```
Comment (dozen12) [inline src/models/cnn_2_5/version/C/model.py:10]: @keechang-choi 여기는 그냥 시비

**Review by dozen12 — CHANGES_REQUESTED** [review]

File: src/models/cnn_2_5/version/C/model.py [hunk 3]
```diff
@@ -5,10 +5,9 @@
 import timm
 from config import CFG
 import torchmetrics
-import torchvision
-
```
Comment (dozen12) [inline src/models/cnn_2_5/version/C/model.py:9]: @keechang-choi 빈 칸을 니 맘대로 하지말라니까?

**Review by dozen12 — CHANGES_REQUESTED** [review]

**Review by dozen12 — APPROVED** [review]
